oauth2/smart/patient-select.php

```diff
@@ -0,0 +1,173 @@
+                <?php if (!empty($errorMessage)) : ?>
+                    <p class="alert alert-warning"><?php echo xlt($errorMessage); ?></p>
+                <?php endif; ?>
+
+                <?php if (count($patients) < 0) : ?>
+                    <p class="alert alert-info"><?php echo xlt("No patients to select"); ?></p>
+                <?php else : ?>
+                <form action="<?php echo $searchAction; ?>" method="GET">
+                    <input class="w-25" name="search[fname]" type="text" class="form-control form-input" placeholder="<?php echo xlt("First Name"); ?>"
+                           value="<?php echo attr($fname); ?>" />
+                    <input class="w-25" name="search[mname]" type="text" class="form-control form-input" placeholder="<?php echo xlt("Middle Name"); ?>"
+                           value="<?php echo attr($mname); ?>" />
+                    <input class="w-25" name="search[lname]" type="text" class="form-control form-input" placeholder="<?php echo xlt("Last Name"); ?>"
+                           value="<?php echo attr($lname); ?>" />
+                    <input type="submit" value="Search" />
+                </form>
+                    <?php if ($hasMore) : ?>
+                <p class="alert alert-info"><?php echo xlt("Too many search results found.  Displaying a limited set of patients.  Narrow your search results through the filters above."); ?></p>
+                <?php endif; ?>
+            </div>
+        </div>
+        <div class="row w-100">
+            <p class="col">
+
+                <table class="table table-striped">
+                    <thead>
+                        <tr>
+                            <th><?php echo xlt("Name"); ?></th>
+                            <th><?php echo xlt("DOB"); ?></th>
+                            <th><?php echo xlt("Sex"); ?></th>
+                            <th><?php echo xlt("Email"); ?></th>
+                            <th></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <?php foreach ($patients as $patient) : ?>
+                        <tr>
+                            <td>
+                                <?php if ($patient['mname']) : ?>
+                                    <?php echo sprintf("%s %s %s", $patient['fname'], $patient['mname'], $patient['lname']); ?>
+                                <?php else : ?>
+                                    <?php echo sprintf("%s %s", $patient['fname'], $patient['lname']); ?>
+                                <?php endif; ?>
+                            </td>
+                            <td>
+                                <?php echo $patient['DOB']; ?>
+                            </td>
+                            <td>
+                                <?php echo $patient['sex']; ?>
+                            </td>
+                            <td>
+                                <?php echo $patient['email']; ?>
```

Comment:
text() here and in all above places (line 91 to here) where are outputting $patient variables